src/WebJobs.Script.Abstractions/Description/FunctionMetadata.cs

```diff
@@ -42,11 +42,6 @@ public FunctionMetadata()
         /// </summary>
         public IDictionary<string, object> Properties { get; }
 
-        /// <summary>
-        /// Gets or sets the optional function execution retry strategy to use on invocation failures.<see cref="Abstractions.Description.RetryOptions"/>.
-        /// </summary>
-        public RetryOptions Retry { get; set; }
```

Comment:
I thought we had told logic apps to use this as a way to specify retry for their dynamically generated functions? I.e. this breaking change might affect them

Reply:
Chatted with LogicApps offline - they do not use Retry. So we should be good.